OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardFragment.java

```diff
@@ -123,7 +121,14 @@ public void onStop() {
         super.onStop();
         if (updateDownloadProgressRunnable != null) {
             handler.removeCallbacks(updateDownloadProgressRunnable);
-            wasDownloadItemVisibleBeforeStopping = downloadsMenuItem.isVisible();
+        }
+    }
+
+    @Override
+    public void onDestroyView() {
+        super.onDestroyView();
+        if (updateDownloadProgressRunnable != null) {
+            handler.removeCallbacks(updateDownloadProgressRunnable);
             updateDownloadProgressRunnable = null;
```

Comment:
I don't think this is needed. Any reason null assigning was done here?

Reply:
@miankhalid We need to create the new instance of the `Runnable` on orientation change, else  not updating the video download progress. So make it null before creating new instance.

Reply:
got it. It would be great if you add this as a comment in the code for future safety.